### PR TITLE
models.py complete. added 0002_users_test_data.py. added line in setup.sh

### PR DESCRIPTION
**Pull request by razashtar93**
I've got some problem when trying to do issue #39
after added the class Users to models.py and ran the command:
pipenv run python manage.py makemigrations AcadeMeData which created the file: 0001_initial,
I created the file 0002_users_test_data and try to run the command:
pipenv run python manage.py migrate
and got the following error:
 
image
image

My assumption is that when I use the class models.User from the library of django every "user" object that created got a user_id 
field which make sense but I can't figure it out the line:  django.db.utils.OperationalError: no such column: AcadeMeData_users.user_id. I googled it without any success ..
I'll be glad to receive your help.

Once I'll figure it out what the problem with the migrations I will move to add test class to
the file test.py and move on to the next model class.
------------------------------------------------------------------
About this PR:
this PR overwrite PR#30 because of rebasing problems.
also added : 
# AcadeMe 
AcadeMe/.idea/
.idea/workspace.xml
.idea
to the .gitignore file - for some reason I still got the workspace.xml as untracked file.

### CODE REVIEW

File: AcadeMeData/models.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+from django.db import models
+from django.conf import settings
+from django.contrib.auth.models import User
```
Comment (Yarboa) [inline AcadeMeData/models.py:3]: Please refer this class never used

**Comment by Yarboa** [comment]
In genereal you should add the migration commands to the setup.sh

**Review by Yarboa — COMMENTED** [review]
If you will delete your 
db.sqlite3 file. ssh to vagarnt cd /vagrant
run command:
pipenv run python manage.py migrate 
It will pass with success, i tried it

Please add to the migration line to setup.sh

Do not forget to add test

**Comment by razashtar93** [comment]
models.py complete
add line in setup.sh
add 0002_users_test_data.py

File: AcadeMeData/models.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+                               blank=True, null=True)  # average professor rating, starts as null
+
+    # for DecimalField goto: https://www.geeksforgeeks.org/decimalfield-django-models/
+
+    def __str__(self):
+        return self.name
+
+
+class Degree(models.Model):
+    Degree_id = models.IntegerField(primary_key=True,
```
Comment (bregman-arie) [inline AcadeMeData/models.py:92]: should be lower case (and apply to all variables)

File: AcadeMeData/migrations/0001_initial.py [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+import django.core.validators
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+    initial = True
+
+    dependencies = [
+        ('auth', '0012_alter_user_first_name_max_length'),
```
Comment (Yarboa) [inline AcadeMeData/migrations/0001_initial.py:13]: '0012_alter_user_first_name_max_length')

Once you are running make migration and you have already exist data in the database, this is automatically created, 

Please delete db.sqlite3 and run the make migration,
Only clean and relevant files are needed,

**Review by Yarboa — COMMENTED** [review]
See, there is a PR alreadt running in Parallel.
I would advise you, only to focus in migration files.
And add a test, that read the users, no more than that,

If now we are going to circle for many days on that

**Comment by Yarboa** [comment]
> See, there is a PR alreadt running in Parallel. I would advise you, only to focus in migration files. And add a test, that read the users, no more than that,
> 
> If now we are going to circle for many days on that

sorry please ignore this one

File: .gitignore [hunk 4]
```diff
@@ -135,5 +135,14 @@ dmypy.json
 nohup.out
 
 # AcadeMeData app
-AcadeMeData/.idea/
+ AcadeMeData/.idea/
+
+# AcadeMe 
+ AcadeMe/.idea/
```
Comment (Yarboa) [inline .gitignore:141]: Did you try this ? 
AcadeMe/**/.idea

**Review by Yarboa — COMMENTED** [review]
I would consider adding a test here.
At lease one or two methods for the ORM classes, to test, WDYT?

File: .gitignore (hunk 4 above)
Comment (Yarboa) [reply .gitignore:141]: see @bregman-arie review to the other PR related to git ignore

Comment (Yarboa) [reply .gitignore:141]: I think this already done in #51 
Please rebase with upstream

File: AcadeMeData/migrations/0001_initial.py (hunk 3 above)
Comment (Yarboa) [reply AcadeMeData/migrations/0001_initial.py:13]: My mistake

**Comment by Yarboa** [comment]
Please rebase with upstream

**Pull request closed without merging**